pkg/controller/certificates/certificate_controller.go

```diff
@@ -79,29 +79,29 @@ func NewCertificateController(
 	csrInformer.Informer().AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc: func(obj interface{}) {
 			csr := obj.(*certificates.CertificateSigningRequest)
-			klog.V(4).Infof("Adding certificate request %s", csr.Name)
+			klog.V(4).InfoS("Adding certificate request", "csr", klog.KObj(csr))
```

Comment:
CSR is an acronym https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments
```suggestion
			klog.V(4).InfoS("Adding certificate request", "CSR", klog.KObj(csr))
```

Reply:
Fixed.